style.css

```diff
@@ -31,10 +110,63 @@ p {
 
 #add-btn {
   padding: 5px 15px;
-  font-size: 15px;
+  font-size: 1.5rem;
+  margin: 0.2rem 0 0 75%;
 }
 
-hr {
-  margin-bottom: 20px;
-  margin-top: 3px;
+#Copy_right {
+  border: 1px solid black;
+  box-shadow: 0.05rem 0.05rem 2rem;
+  font-size: 1.5rem;
+  padding: 0.5rem;
+  margin: 4.5rem 1rem 0.5rem 1rem;
+}
+
+.contact_info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 3rem 0 3rem 0;
+}
+
+.contact_info ul {
+  align-self: flex-start;
+  padding-left: 35%;
+  font-size: 1.5rem;
+}
+
+.contact_info p {
+  font-size: 1.5rem;
+  padding-bottom: 1.5rem;
+}
+
+.book_lists {
+  display: flex;
+  justify-items: center;
+  padding: 1rem;
+  gap: 0.2rem;
+  width: 35rem;
+  margin: 0 3rem 0 35rem;
+  border: 3px solid black;
+}
```

Comment:
- [ ] All the classes 'book_lists' seem to stay in one place and that does not let them be responsive. 
![Screen Shot 2022-02-09 at 16 35 17](https://user-images.githubusercontent.com/67211919/153302491-4d26129d-a5e2-42b1-8313-cf1e0284feb3.png)